### ISSUE TITLE
[Bug] Getting logits not supported for batch size > 1 in Qwen2 0.5B

### ISSUE THREAD

**Issue by hiwudery** [airockchip/rknn-llm#406]
**Steps to Reproduce**
Load Qwen2 0.5B model using RKNN-LLM
Set batch size to 2
Attempt to retrieve logits output
Error occurs

**Expected Behavior**
Should be able to get logits output with batch size > 1, or at minimum receive clear documentation that this limitation exists.
Actual Behavior
Application throws error: "rkllm: Getting the logits is not supported for multiple batches."

**Environment**
Model: Qwen2 0.5B
Batch Size: 2
Framework: RKNN-LLM 1.2.2

**Questions**
Is this a known limitation of the current implementation?
Are there plans to support logits extraction for batch size > 1?
Is there a workaround available?
